site/docs/v1/tech/apis/system.adoc

```diff
@@ -357,7 +359,7 @@ link:/docs/v1/tech/apis/authentication#api-key-authentication[icon:lock[type=fas
 === Response
 The JSON response from the Status API is complex and subject to change. The only exception is the `version` key.
 
-`version` will not change and will be returned as below.  As a reminder, an API key is required to obtain this value.
+`version` will not change and will be returned as below. As a reminder, unless explicitly allowed from `localhost`, an API key is required to obtain this value.
```

Comment:
I think it's easier to follow with the caveat at the end than in the middle. I'm on the fence about whether to include the link again.

```suggestion
`version` will not change and will be returned as below. As a reminder, an API key is required to obtain this value unless explicitly allowed from `localhost` as link:/docs/v1/tech/reference/configuration#[documented here].
```